### DISCUSSION TITLE
Trunk to trunk transfer with a twist

### DISCUSSION THREAD

**Discussion by mostafa0017** [mwolff44/pyfreebilling#204] [Q&A]
I have a situation where I need to supply a CRM platform with a number for inbound and outbound calls. The number is provisioned via a trunk from Yeastar TG800 VOIP GSM gateway. The platform uses SIP URIs.

Can this proxy be used to do the following ?

When an inbound call reaches the GSM gateway, it should be transferred via a trunk. This call should be transfered to the CRM platform that uses (expects) calls on a certain SIP URI.

When an outbound call is made from the CRM dialer to "BBBBB". This call should be forwarded/transferred to the GSM gateway using the Trunk so that the gateway continues to call "BBBBB" using the relevant SIM card associated with the VOIP account that was used to send the call to the gateway by the system (SIM card association to VOIP accounts is done inside the GSM gateway already).

And if not, what a system that can do something like this would be called ? And any recommendations ?

**Comment by mwolff44** [comment]
Hi,
Yes, you can use PK-SBC for that.
Mathias
